### ISSUE TITLE
🐛 BUG: missing border radius on the card

### ISSUE THREAD

**Issue by sshanzel** [dailydotdev/daily#1411]
### What went wrong? 🤔

The card itself has a border-radius but the focus highlight does not display the same. It is likely due to some container or wrapper on the card that does not share the same border-radius.

![image](https://github.com/user-attachments/assets/a6fbb732-ad0e-4d10-890c-b9f392662b17)

### Expected Behavior

Should have the same amount as the card.

### Steps to Reproduce Issue

```bash
1. Go to the feed
2. Click on any post to open a modal
3. Click escape on your keyboard
4. See the issue
```

### Solution Proposed

Share the same border-radius

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by Ahmad-Zahid** [comment]
@sshanzel Hi, can I take this up?

**Comment by Ankita-Nandkumar-Patil** [comment]
@sshanzel I have checked and tried to fix the issue,
please review the PR and merge
https://github.com/dailydotdev/apps/pull/3574

**Comment by sshanzel** [comment]
@Ahmad-Zahid sorry I did not get the chance to reply to you last week. It looks though that @Ankita-Nandkumar-Patil has worked on it already, so let's see how it went 👀 

Thank you both for the interest.

**Comment by Ankita-Nandkumar-Patil** [comment]
hey @sshanzel Thank you for assigning,
please check the PR, and let me know if anything required further
https://github.com/dailydotdev/apps/pull/3574

**Comment by sparshcodes** [comment]
Is this still open?

**Comment by sparshcodes** [comment]
@sshanzel i have fixed the issue, now it will only work for article cards.
Please review #3662

**Comment by SuyashShukla0007** [comment]
@sshanzel  is it still open?

**Comment by satvik2131** [comment]
@SuyashShukla0007 , nope it's solved . You can check it when running gitpod

**Comment by sshanzel** [comment]
Yeah, this should've been fixed a week ago by another OSS contributor as well.
